scripts/benchmark.ts

```diff
@@ -0,0 +1,207 @@
+/// <reference lib="deno.unstable" />
+
+// Run with: deno run -A --unstable ./scripts/benchmark.ts --path http://127.0.0.1:4512 --read-concurrency 128 --write-concurrency 32 --size 10000 --consistency-check-interval 10
```

Comment:
Nice benchmark, can you move this file in `cli/bench`? Also is it possible to use `Deno.bench` here?

Reply:
> can you move this file in cli/bench? 

This benchmark depends on a running `denokv` service. Not sure if it fits in `cli/bench`...

> Also is it possible to use Deno.bench here?

Here we are benchmarking more about throughput rather than latency - is `Deno.bench` suitable for measuring highly concurrent workloads?

Reply:
Ah I forgot this was the `denokv` repo 😅